components/raftstore/src/store/peer.rs

```diff
@@ -5305,6 +5357,10 @@ where
         !has_overlapped_reads && !has_overlapped_writes
     }
 
+    pub fn set_leader_lease_expire(&mut self) {
+        self.leader_lease.expire();
+    }
+
```

Comment:
I will change `self.leader_lease.expire()` to `self.leader_lease.expire_remote_lease()` there. @Connor1996

Reply:
use getset::MutGetters to replace it.